Plugins/Source/KISAddonCursor.cs

```diff
@@ -89,6 +82,17 @@ sealed class KISAddonCursor : MonoBehaviour {
     Cursor.visible = false;
   }
 
+  void Awake() {
+    hintWindowStyle = new GUIStyle {
```

Comment:
Disposing is a problem anyways. Every instantiation of the cursor will create new instance of `GUIStyle`, and old one will be just abandoned. Though, I don't think this kind of resource needs disposing.

That said, converting into instance member should be OK.